opencti-platform/opencti-graphql/src/schema/stixRefRelationship.ts

```diff
@@ -54,31 +54,31 @@ export const ABSTRACT_STIX_NESTED_REF_RELATIONSHIP = 'stix-nested-ref-relationsh
 
 export const INPUT_OPERATING_SYSTEM = 'operatingSystems';
 export const INPUT_SAMPLE = 'samples';
-export const INPUT_CONTAINS = 'contains';
+export const INPUT_CONTAINS = 'containsObservable';
 export const INPUT_RESOLVES_TO = 'resolvesTo';
 export const INPUT_BELONGS_TO = 'belongsTo';
-export const INPUT_FROM = 'from';
-export const INPUT_SENDER = 'sender';
-export const INPUT_TO = 'to';
-export const INPUT_CC = 'cc';
-export const INPUT_BCC = 'bcc';
+export const INPUT_SENDER = 'emailSender';
```

Comment:
I thought we talk about pushing the objectXXX convention

Reply:
the convention objectXXX produces the change `contains` -> `objectContains`
but `objectContains` was an old filter key [still renamed on the fly](https://docs.opencti.io/latest/reference/filters-migration/) to `objects`, so we can't use objectContains

one thing leading to another, we thought it would be better to prefix with the entity they're used for as per the schema description. It's the case for a lot of other keys.